<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectionWgt</class>
 <widget class="QWidget" name="DirectionWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">DirectionWgt</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>45</y>
     <width>360</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_currPos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>85</y>
     <width>240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>40</height>
    </size>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>999999</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>82</y>
     <width>105</width>
     <height>46</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>105</width>
     <height>46</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>105</width>
     <height>46</height>
    </size>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="buttons" stdset="0">
    <string notr="true">btn_dlg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(228, 231, 236);</string>
   </property>
   <property name="text">
    <string notr="true">XXX</string>
   </property>
   <property name="paddings_left" stdset="0">
    <string notr="true">l_10</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>134</y>
     <width>355</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <item>
     <widget class="QPushButton" name="btn_backMove">
      <property name="minimumSize">
       <size>
        <width>106</width>
        <height>46</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>46</height>
       </size>
      </property>
      <property name="text">
       <string>反向</string>
      </property>
      <property name="buttons" stdset="0">
       <string notr="true">btn_dlg</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="spinBox_pos">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_frontMove">
      <property name="minimumSize">
       <size>
        <width>106</width>
        <height>46</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>46</height>
       </size>
      </property>
      <property name="text">
       <string>正向</string>
      </property>
      <property name="buttons" stdset="0">
       <string notr="true">btn_dlg</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
